### ISSUE TITLE
Error 502

### ISSUE THREAD

**Issue by grzegorzsurowiec** [kasmtech/workspaces-issues#305]
Hello everyone.
I tried to solve the problem myself without success.
I used two different types of network, two machines (pc, and proxmox).
3 different operating systems. And I still have the same error.
Kasm installs correctly.
I can log in to the panel but I can't start any instance.
The console shows errors like:
[HTTP/1.1 502 Bad Gateway 24ms]

**Comment by grzegorzsurowiec** [comment]
The issue is with version 1.12.0.
Version 1.9.0 works fine.

**Comment by j-travis** [comment]
We are going to need more info to help you out.

- How are you connecting to your Kasm Deployments? : Browser, browser version, os , osversion

- Are you using any forward or reverse proxy in between you or Kasm?

- Take a look at our troubleshooting guide (https://kasmweb.com/docs/latest/guide/troubleshooting.html#client-side-issues) . do you see any errors in your browser developer console?

- What are the details of the Os/Distro/Architectures you've tried

**Comment by teepee1234** [comment]
Hello,
I am noticing the same issue with 1.12.0.  502 bad gateway in console.  Used uninstall instructions and then installed 1.11.0, everything works great.
How are you connecting to your Kasm Deployments? : **Browser Chome 108.0.5359.125**
Are you using any forward or reverse proxy in between you or Kasm? **no**
do you see any errors in your browser developer console? **Yes 502 bad gateway**
Running on Ubuntu 22.0.4 x86_64

Great product!
Thanks

**Comment by grzegorzsurowiec** [comment]
Hi.
Sorry it took so long to reply but it was Christmas.
I checked everything a few more times.

The browser doesn't matter (I checked chrome and firefox).
It doesn't matter from where I enter the panel - I tried from ubuntu itself on which kasm is installed, as well as from the local network.
No problem with the previous version.
In version 1.12, an error is shown in the console at the very beginning: "Cannot parse user info. No user info stored"
502 errors are also shown in the old version, but it does not prevent the containers from firing correctly.

**Comment by grzegorzsurowiec** [comment]
In the old version, when logging in for the first time, an error is shown:
""Constants.js:37 Cannot parse user info. No user info stored
m @ Constants.js:37
Updated user info: { XXXX"

In the new version after first login an error is shown:
"Constants.js:37 Cannot parse user info. No user info stored
m @ Constants.js:37
null:1          Failed to load resource: the server responded with a status of 404 (Not Found)
moment.js:287 Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: None, _f: undefined, _strict: undefined, _locale: [object Object]
Error
    at Function.createFromInputFallback (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3418477)
    at Ir (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3442869)
    at Vr (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3446201)
    at Qr (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3446068)
    at Hr (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3445787)
    at Wr (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3446559)
    at Kr (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3446593)
    at a (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3415353)
    at cell (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:674822)
    at Fs (https://10.0.0.40/index.bundle.js?2e989b9d6af53ef3b82b:2:3849075)
k @ moment.js:287
(anonymous) @ moment.js:319
Ir @ moment.js:2550
Vr @ moment.js:2993
Qr @ moment.js:2976
Hr @ moment.js:2943
Wr @ moment.js:3037
Kr @ moment.js:3041
a @ moment.js:16
cell @ AdminUser.js:191
Fs @ react-dom.production.min.js:167
wl @ react-dom.production.min.js:290
yu @ react-dom.production.min.js:280
_u @ react-dom.production.min.js:280
gu @ react-dom.production.min.js:280
lu @ react-dom.production.min.js:273
Ga @ react-dom.production.min.js:127
ru @ react-dom.production.min.js:266
enqueueSetState @ react-dom.production.min.js:145
_.setState @ react.production.min.js:12
(anonymous) @ Table.js:331
v @ runtime.js:65
(anonymous) @ runtime.js:303
F.forEach.e.<computed> @ runtime.js:117
n @ Table.js:337
(anonymous) @ Table.js:337
Promise.then (async)
n @ Table.js:337
(anonymous) @ Table.js:337
i @ Table.js:337
a.value @ Table.js:337
h @ lodash.js:10401
y @ lodash.js:10450
b @ lodash.js:10438
setTimeout (async)
Lo @ lodash.js:6710
g @ lodash.js:10409
v @ lodash.js:10478
value @ methods.js:364
value @ lifecycle.js:38
vl @ react-dom.production.min.js:261
yl @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
(anonymous) @ react-dom.production.min.js:283
Eu @ react-dom.production.min.js:281
lu @ react-dom.production.min.js:273
Ga @ react-dom.production.min.js:127
uu @ react-dom.production.min.js:274
Me @ react-dom.production.min.js:52
Yn @ react-dom.production.min.js:109
Vt @ react-dom.production.min.js:74
Ht @ react-dom.production.min.js:73"

**Comment by grzegorzsurowiec** [comment]
All version < 1.12 work ok
I tested several time on diffrent os and machines.
Today i downgrade from 1.12 to 1.11 (uninstall and install) and it works.

**Comment by j-travis** [comment]
Can you see if this is your issue?
https://github.com/kasmtech/workspaces-issues/issues/294

**Comment by grzegorzsurowiec** [comment]
I tested several scenarios.
1. Clean install - 1.12 - works with override https://github.com/kasmtech/workspaces-issues/issues/294.
2. Clean install - 1.9.0 - everything works fine.
3. Clean install  - 1.11 - everything works fine.
4. Update 1.9.0 > 1.11 - everything works fine.
5. Update 1.11 > 1.12  - everything works fine.
6. Update 1.9.0 > 1.11 > 1.12 - everything works fine.

Explanation - a clean install means reinstalling the entire system (formatting disks).

I tested it on two environments - baremetal and proxmox.
In both cases, the situation is analogous.


I won't write how much time I spent on it :)
